### ISSUE TITLE
False-Positive: Malicious code in coloramas

### ISSUE THREAD

**Issue by manuel-cohere** [owasp-dep-scan/dep-scan#341]
### PURL of wrongly matched component

pypi:coloramas

### Depscan findings

I don't get the same output with that command.

**Comment by prabhu** [comment]
@manuel-cohere https://osv.dev/vulnerability/MAL-2024-4924

**Comment by manuel-cohere** [comment]
I apologise for the incomplete report.

This is what I'm getting:
``` JSON
        {
            "bom-ref": "MAL-2024-4924/pkg:pypi/colorama@0.4.6",
            "id": "MAL-2024-4924",
            "source": {},
            "ratings": [
                {
                    "score": 10.0,
                    "severity": "critical",
                    "vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H",
                    "method": "CVSSv31"
                }
            ],
            "cwes": [],
            "description": "# Malicious code in coloramas (PyPI)\n\n---\n_-= Per source details. Do not edit below this line.=-_",
            "recommendation": "",
            "advisories": [],
            "analysis": {
                "state": "in_triage",
                "detail": "Dependency Tree: [\"pkg:pypi/ariadne-codegen@0.12.0\", \"pkg:pypi/click@8.1.7\", \"pkg:pypi/colorama@0.4.6\"]"
            },
```

It is flagging `pypi/ariadne-codegen@0.12.0`, but that package depends on `colorama` not `coloramas`

**Comment by prabhu** [comment]
Thank you! This is fixed by #346
